pennylane/transforms/tape_expand.py

```diff
@@ -18,13 +18,13 @@
 import warnings
 
 import pennylane as qml
-from pennylane import math
+from pennylane.math import get_trainable_indices
```

Comment:
```suggestion
from pennylane.math import get_trainable_indices, requires_grad
```

Reply:
Actually, since there's multiple, I'm just going to import `math`.